docs/markdown/snippets/wrap.md

```diff
@@ -0,0 +1,18 @@
+## Automatic fallback to `cmake` subproject
+
+CMake subprojects have been supported for a while using `cmake.subproject()`
+module method. However until now it was not possible to use a CMake subproject
+as fallback in a `dependency()` call.
+
+A wrap file can now specify the method used to build it by setting `method=cmake`
```

Comment:
I don't think `method=cmake` is what you want here. I think it might better to use `by setting the method key`

Reply:
done